src/web/lib/contributions.ts

```diff
@@ -149,3 +150,20 @@ export const getArticleCountConsent = (): Promise<boolean> => {
         });
     });
 };
+
+export const withinLocalNoBannerCachePeriod = (): boolean => {
+    const item = window.localStorage.getItem(NO_BANNER_UNTIL_LATER_KEY);
+    if (item && !Number.isNaN(parseInt(item, 10))) {
+        const noBanner = parseInt(item, 10) > Date.now();
+        if (!noBanner) {
```

Comment:
can we flip the logic so this isn't a double negative? or just change NoBanner to IsCached